.github/workflows/renovate-vault.yml

```diff
@@ -16,9 +16,13 @@ on:
   schedule:
     - cron: '30 4,6 * * *'
 
+permissions:
+  contents: read
+  id-token: write
+
 jobs:
   call-workflow:
-    uses: rancher/renovate-config/.github/workflows/renovate.yml@modigest
+    uses: rancher/renovate-config/.github/workflows/renovate-vault.yml@release
```

Comment:
We may need to get [this](https://github.com/rancher/renovate-config/commit/a8b0c64e758dc332454d5e9203ab24060d676b35) change merged into release, before this PR can be merged.